tiatoolbox/tools/patchextraction.py

```diff
@@ -221,22 +220,22 @@ def filter_coordinates_fast(
 
         Args:
             mask_reader (:class:`.VirtualReader`): a virtual pyramidal reader of the
-              mask related to the WSI from which we want to extract the patches.
+                mask related to the WSI from which we want to extract the patches.
             coordinates_list (ndarray and np.int32): Coordinates to be checked
-              via the `func`. They must be in the same resolution as requested
-              `resolution` and `units`. The shape of `coordinates_list` is (N, K)
-              where N is the number of coordinate sets and K is either 2 for centroids
-              or 4 for bounding boxes. When using the default `func=None`, K should be
-              4, as we expect the `coordinates_list` to be refer to bounding boxes in
-              `[start_x, start_y, end_x, end_y]` format.
-            coord_resolution (float): the resolution value at which coordinates_list are
-              generated.
-            coord_resolution (str): the resolution unit at which coordinates_list are
-              generated.
-            mask_resolution (floar): resolution at which mask array is extracted. It is
-              supposed to be in the same units as `coord_resolution` i.e.,
-              `coord_units`. If not provided, a default value will be selected based on
-              `coord_units`.
+                via the `func`. They must be in the same resolution as requested
+                `resolution` and `units`. The shape of `coordinates_list` is (N, K)
+                where N is the number of coordinate sets and K is either 2 for centroids
+                or 4 for bounding boxes. When using the default `func=None`, K should be
+                4, as we expect the `coordinates_list` to be referred to bounding boxes
+                in `[start_x, start_y, end_x, end_y]` format.
+            coordinate_resolution (float): the resolution value at which
+                coordinates_list are generated.
```

Comment:
\` coordinate_list \`

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/281/commits/d067b29e6e28646aa6552de10919f37690f32c3e